app/views/pages/statistics.ejs

```diff
@@ -1,319 +1,373 @@
-<%
-  const totalPeopleInContactWithInfected = locals.inContactWithInfectedStats.total;
-  const totalInfectedPeopleWithSymptoms = locals.infectedStats.symptomStats.total;
-  const totalPeopleWithSymptoms = locals.allSymptomsStats.symptomStats.total;
-  function numberWithSpaces(x) {
-    return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, " ");
-  }
-%>
-<!DOCTYPE html>
-<html lang="<%= locals.htmlLang %>">
-  <%- include('partials/head') -%>
-   <body>
-    <%- include('partials/disclaimer') -%>
-    <%- include('partials/menu') -%>
-    <%- include('partials/hamburger-menu') -%>
-    <div id="page-wrapper">
-      <section id="main" class="wrapper">
-        <div class="inner">
-          <h1><%= __('Statistics') %></h1>
-          <form method="GET" action="/api/reports/reports.csv" style="margin-bottom: 0">
-            <ul class="actions">
-              <li>
-                <input type="submit" value="<%= __('Download the dataset as CSV') %>">
-              </li>
-            </ul>
-          </form>
-          <h2 class="stat-heading"><%= __('COVID-19 infected') %></h2>
-          <p><%= __('In total') %> <%= numberWithSpaces(totalInfectedPeopleWithSymptoms) %> <%= __('people have reported that they have tested positive for COVID-19 and experience symptoms.') %></p>
-          <div class="stat-layout">
-            <div class="stat-cell">
-              <h3 class="stat-heading"><%= __('Symptoms') %></h3>
-              <canvas id="infectedSymptomsChart"></canvas>
-            </div>
-            <div class="stat-cell">
-              <h3 class="stat-heading"><%= __('Test results') %></h3>
-              <p><%= __('In total') %> <%= numberWithSpaces(totalPeopleInContactWithInfected) %> <%= __('people have reported that they have been tested for COVID-19.') %></p>
-              <div class="stat-layout">
-                <div class="stat-cell">
-                  <canvas id="testResultStats"></canvas>
-                </div>
-              </div>
-            </div>
-          </div>
+<% const totalPeopleInContactWithInfected =
+locals.inContactWithInfectedStats.total; const totalInfectedPeopleWithSymptoms =
```

Comment:
A bit weird formatting here 😅

Reply:
It's prettier, can't really help it or let them ignore it.